library/modules/Gui.cpp

```diff
@@ -1360,6 +1368,100 @@ DFHACK_EXPORT void Gui::writeToGamelog(std::string message)
     fseed.close();
 }
 
+namespace
```

Comment:
I'm not sure if we have a policy around this, but most other DFHack code uses static functions instead of anonymous namespaces.